src/Camille.RiotGames/gen/configspec.json

```diff
@@ -3,6 +3,21 @@
     "type": "string",
     "desc": "Riot Games API key."
   },
+  "apiUrl": {
+    "type": "string",
+    "desc": "URL to call for the Riot Games API. Include {0} for the region, if you want it in the URL - Not necessary for RegionConfig.inUrlAsRegionQueryParameter.",
```

Comment:
"Not necessary if apiCallRegionConfig is RegionConfig.InUrlAsRegionQueryParameter"